src/3rdparty/walkontable/src/viewport.js

```diff
@@ -64,17 +64,17 @@ class Viewport {
   getWorkspaceWidth() {
     const { wot } = this;
     const { rootDocument, rootWindow } = wot;
+
+    if (wot.getSetting('preventOverflow')) {
+      return outerWidth(this.instance.wtTable.wtRootElement);
+    }
+
     const trimmingContainer = this.instance.wtOverlays.leftOverlay.trimmingContainer;
     const docOffsetWidth = rootDocument.documentElement.offsetWidth;
```

Comment:
if set the preventOverflow, we can return the call in advance without calc the documentElement.offsetWidth,
which will cause the reflow